src/test/java/core/basesyntax/strategy/handlers/BalanceOperationTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.strategy.handlers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.handlers.FruitTransactionException;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationTest {
+    private static final String FRUIT_NAME = "strawberry";
+    private static final int FRUIT_COUNT = 20;
+
+    @Before
+    public void before() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void addNew_ok() {
+        BalanceOperation balanceOperation = new BalanceOperation();
```

Comment:
let's make this operation a class field, as you have done before. Fix similar issue in all places